data/json/recipes/other/vehicles.json

```
@@ -1890,7 +1892,8 @@
     "difficulty": 2,
     "autolearn": true,
     "time": "30 m",
+    "batch_time_factors": [ 60, 1 ],
     "using": [ [ "3d_printing_standard", 30 ] ],
```

Time and tool use on this recipe is also connected to how many fibers it uses, so we can bump those down too. :D
```suggestion
    "time": "20 m",
    "batch_time_factors": [ 60, 1 ],
    "using": [ [ "3d_printing_standard", 20 ] ],
```